src/MonoMod.UnitTest/MonoMod.UnitTest.csproj

```diff
@@ -11,12 +11,14 @@
 
     <TargetFrameworks>net46;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
     <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 16.6.0 And $(NETCoreAppMaximumVersion) &gt;= 5.0">$(TargetFrameworks);net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.0.0 And $(NETCoreAppMaximumVersion) &gt;= 6.0">$(TargetFrameworks);net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.0.0 And $(NETCoreAppMaximumVersion) &gt;= 6.0">net6.0;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
Should probably include a comment explaining why this is inconsistent. That being said, I don't think we support old SDKs anymore anyway, so all of these conditionals can probably be removed.